### PR TITLE
PR - Visitor can see list of articles

### PR DESCRIPTION
**Pull request by Ehugo2000**
[Visitor can view all listed articles API ](https://www.pivotaltracker.com/story/show/176031773)
### User Story
```
As an API
In order to provide the client with a list of articles
I would like to have a database of demonetized articles
```
## Changes proposed in this pull request:
* Generate Articles Controller and Article model
* Adding Fbot for articles and content
* Adding tests for API requests 
* Creating Active Model Serializers to sanitize JSON objects
## What I have learned working on this feature:
* Postman functions
* How to work with AMS
## Packages/Gems added
* Active Model Serializers

### CODE REVIEW

File: db/migrate/20201210090008_create_articles.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class CreateArticles < ActiveRecord::Migration[6.0]
+  def change
+    create_table :articles do |t|
+      t.string :title
+      t.text :lead
+      t.text :body
```
Comment (emtalen) [inline db/migrate/20201210090008_create_articles.rb]: looking at your PT story you shouldn't have body

File: app/serializers/articles_index_serializer.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class ArticlesIndexSerializer < ActiveModel::Serializer
+  attributes :id, :title, :lead, :body, :created
```
Comment (emtalen) [inline app/serializers/articles_index_serializer.rb]: According to your PT story you are not returning body and that is also the assumption that your frontend team has. The serializer is for not sending unused data.

**Review by emtalen — CHANGES_REQUESTED** [review]
Either have a team discussion and change the pt story regarding the body of the article on the index action or otherwise that is out of scope for this feature.

File: db/migrate/20201210090008_create_articles.rb (hunk 1 above)
Comment (salindholm) [reply db/migrate/20201210090008_create_articles.rb]: Okey, this is now fixed!

**Review by emtalen — APPROVED** [review]

**Pull request merged**